.gitignore

```diff
@@ -18,8 +18,5 @@ NashornProfile.txt
 /src/utils/LogCompilation/target/
 /.project/
 /.settings/
-/.project
-/.classpath
```

Comment:
`.project` is repeated so that is fine to remove, but why are you removing the other two? `.cproject` sounds like it could come from CDT.

Reply:
With the Shared Workspaces removed, the Workspace created no longer exists in the JDK's top level source directory. This means the .gitignore entries are now obsolete, and can be removed, since they only ignore .cproject and .classpath in the top level source directory. They were added way back when I introduced support for Eclipse, and was less familiar with how multiple different JDK builds are contained

Reply:
Side tangent: How are the Workspaces for the other IDEs set up? Are they contained entirely in the ide subdirectory in the build directory or do they end up leaving their metadata in the top level directory like the Shared Workspaces used to do? I plan to improve Eclipse support over time, and these details might potentially be important to that effort

Reply:
Ok, it's fine for integration then.

As for other IDEs: there is very little coordination on this; most IDE integrations are like barely kept alive by some enthusiast. I would really love to see some better grip on this, unifying IDE support and making it more of a first-class citizen of the build, but this has unfortunately kept being pushed down the prio list. :-(

If you feel so inclined, please try out the different IDEs supported. The very first step, I believe, is to have proper documentation in docs/ide.md. Even if we have a process that requires several manual steps, it is better to have it formalized and documented in a single place. Then it can form the basis for a more automated approach.

Reply:
Sorry I think I may not understand what you mean by unifying IDE support "and making it more of a first class citizen of the build", do you mean a central Setup IDE function in make of some sort, much like SetupNativeCompilation? I don't see how that might be possible given how each IDE has wildly varying requirements and save data formats

Reply:
I think the most confusing part is how different all IDE setup procedures are. The unification I'd like to see is rather like `make ide-setup IDE=vscode AREA=hotspot,jdk` or something like that. That is, a clear and easy to understand way for the user to call a make target to create an environment for their IDE of choice. I realize that the actual implementation will be very varying for different IDEs, but I also think many share similar parts (like creating e.g. an xml file describing the project), and that there certainly are room for even some unification in that part.